lib/screens/creat_group/create_group_screen.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import './widgets/checkbox_list_tile.dart';
+
+class CreateGroupScreen extends StatelessWidget {
+  final String name, role;
+
+  CreateGroupScreen(this.name, this.role);
```

Comment:
1. Convert it into a stateful widget
1. If you think about it doesn't make sense to get a name and role with help of constructor :innocent: